electrum/gui/qt/console.py

```diff
@@ -15,6 +15,10 @@
 
 from .util import MONOSPACE_FONT
 
+# sys.ps1 and sys.ps2 are only declared if an interpreter is in interactive mode.
+sys.ps1 = '>>> '
+sys.ps2 = '... '
```

Comment:
In practice these values would never be defined when using the Qt python console, as presumably Qt would not get run from an interactive interpreter. What are you trying to do by setting these -- is the main purpose just factoring out magic values into named variables/constants?

Reply:
I don't really like the way the prompts were before. The primary prompt '>>> ' was an argument of def __init__() and then was assigned to a property of Console. The other one '... ' wasn't defined anywhere.

Although not very useful, you could now change ps1 and ps2 if you like. I just find it cleaner to use the same variables as in the Python interpreter.

Reply:
`sys.ps1` and `sys.ps2` are defined when using the code module.

Example file `interact.py`:
```
#!/usr/bin/env python3
import code

code.interact()
```

```
$ ./interact.py 
Python 3.8.5 (default, Jul 28 2020, 12:59:40) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
(InteractiveConsole)
>>> import sys
>>> sys.ps1
'>>> '
>>> sys.ps2
'... '
>>> sys.ps1 = '+++ '
+++ sys.ps2 = '!!! '
+++ def f(x):
!!! 
```